Triaging-process.md

```diff
@@ -0,0 +1,57 @@
+
+    ||||||
+    |--- |--- |--- |--- |--- |
+    ||Severity|Severe (blocks or represents a primary user journey – e.g. end user playing a lesson, developer installing Oppia)|Moderate severity (blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log)|Minor (neither of the others, relates to “polish”)|
+    |Users Affected|||||
+    |(roughly) 60+%||High|Medium|Low|
+    |(roughly) 5-60%||High|Low|Low|
+    |(roughly) < 5%||Medium|Low|Low|
+
+    <p style="text-align:center; font-size:12px"> Table: How “impact” is determined </p>
+
+    - ‘Work’ defines how hard it would be to solve/resolve a particular issue. This is generally correlated with the amount of ambiguity in the task. We use the following rough system for assigning this label:
+        - Solution is clear and broken into good-first-issue-sized chunks – assign **Work: Low**
+        - Otherwise, if the means to find the solution is clear – assign **Work: Medium**
+        - Everything else – assign **Work: High**
+
+5. Once the appropriate ‘Impact’ and ‘Work Required’ labels has been added, determine its severity and assign the fitting priority label based on its impact (and possibly the work required label)
+    - If the issue is a high-impact bug, put it in the high-priority bucket and prioritise it within the team.
+    - For the priority bucket, team leads should include “important” issues – typically, a selection of high-impact enhancements and low-work medium-impact bugs – while keeping the size of that bucket about 2-3 times the projected stable number of team members. (Note: This exercise might also help with determining how large the team itself needs to be.)
+    - Other issues go in the ‘backlog’ bucket.
+    - Any issue with an external dependency that the team cannot resolve by itself (eg. missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified on the ‘Issue specific status’ field (using free-text). Team leads should make sure an issue spends as little time here as possible.
+
+6. Determine if the issue is potentially a good first issue
+    - If the issue seems like something a new contributor to Oppia can take up, make sure to add enough information in the issue description for a new contributor to start work on that issue then ask Sandeep or Vojta to add the Good First Issue label. (Note: typically these are “Work: Low” issues.)
+        - For such issues, also assign purple “technology: TypeScript”, “technology: Apache Beam”, “technology: Python”, etc. labels as appropriate. This helps new contributors identify issues that they would find approachable.
```

Comment:
What type of issues? (Good first issues?)

Reply:
Yes these refer to good first issues. Should I explicitly mention that, as they are already nested under a related point?

Reply:
Just to rule out any possibility of confusion. change `For such issues` to `For such good first issues`